changelog.md

```diff
@@ -6,13 +6,13 @@ SciReactUI Changelog
 --------------------
 
 ### Added
-- 
+- SciReactUIProvider to use ThemeProvider and BrowserRouter.
```

Comment:
Maybe highlight this as a breaking change, as it won't work without the <BrowserRouter>. And bump the version to 0.2.0 (also in packages.json)